src/index.html

```diff
@@ -7,8 +7,54 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo_link">
+        <img src="images/logo.png" alt="Moyo logo"
+        class="logo_img">
```

Comment:
please fix the code styling according to the checklist. fix all places in HTML file
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/bd54e510-42bd-4568-bced-347d276869e4)